mk/config.mk

```diff
@@ -32,3 +32,13 @@ CFG_TEE_TA_LOG_LEVEL?=1
 #   marck/check heap feature
 #   Enabling this could decrease efficiency
 CFG_TEE_CORE_USER_MEM_DEBUG?=1
+
+
+# PRNG Configuration:
+#
+# If CFG_WITH_PRNG_SOFTWARE is enabled, crypto provider provided
```

Comment:
Don't forget this comment is you change according to Jerome's proposal about renaming the variable.